auditorium/views/account.js

```diff
@@ -15,102 +15,120 @@ function view (state, emit) {
         </li>
       `
     })
+  var accountHeader = html`
+    <div class="row">
+      ${__('You are logged in as')}
```

Comment:
This will be easier to translate if it's a single string containing the markup (word order might be different in other languages):

```
${raw(__('You are logged in as <strong>operator</strong>'))}
```

Calling `raw` allows for the used string to contain HTML fragments.

Reply:
Yes but in this case it somehow made the docker-compose halting the site loading - could not figure out why